go/test/endtoend/vreplication/config_test.go

```diff
@@ -142,6 +142,22 @@ create table nopk (name varchar(128), age int unsigned);
         "sequence": "customer_seq"
       }
     },
+    "customer_names": {
```

Comment:
You seem to be in the "plural names for tables" camp. The naming of these new tables is inconsistent with that of existing tables.

Reply:
It's already a mix (e.g. orders), but fine. I don't care and if it makes others happy then I'm good with it. :-)

Reply:
Done.